articles/quickstart/spa/react/02-calling-an-api.md

```diff
@@ -17,6 +17,10 @@ useCase: quickstart
 ---
 <!-- markdownlint-disable MD002 MD034 MD041 -->
 
+:::note
+For a deep dive into making secure calls to an API from React, visit the [Call a Protected API from React](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#call-a-protected-api-from-react/). This guide provides you with additional details, such setting up a sample Express API server and getting test access tokens from the Auth0 Dashboard. 
```

Comment:
```suggestion
For a deep dive into making secure calls to an API from React, visit the [Call a Protected API from React](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#call-a-protected-api-from-react/). This guide provides you with additional details, such setting up a sample Express API server and getting test access tokens from the Auth0 Dashboard. 
```
```suggestion
Visit the ["Integrate React with an API Server"](https://developer.auth0.com/resources/guides/spa/react/basic-authentication#integrate-react-with-an-api-server) section of the ["React Authentication By Example"](https://developer.auth0.com/resources/guides/spa/react/basic-authentication) guide for a deep dive into calling a protected API from React. This guide allows you to set up a sample API server using a backend technology of your choice, effectively creating a full-stack application.
```

Reply:
Hello! I'm going to push changes in Angular that match this new note ^ @dan-auth0 could you possibly remove the quotation marks from the page/section titles? None of the other entries have this and we honestly don't use quotes around articles in Docs. I couldn't add a suggestion to the batch.

Reply:
@avanscoy Done 💪